### PR TITLE
Add Bumpversion and Publishing package releases using GitHub Actions 

### PR DESCRIPTION
**Pull request by victoria-cherkas**

### CODE REVIEW

**Comment by cosunae** [comment]
launch jenkins

**Comment by victoria-cherkas** [comment]
launch jenkins

**Comment by victoria-cherkas** [comment]
One of these checks will not pass because i cannot reupload the wheel / tar files to testpypi - see https://test.pypi.org/help/#file-name-reuse

But the file that is there was created successfully when I bumped version. (I have since unbumped the version back 0.1.19 given that we should just run bumpversion from main branch after PR merges.

File: .github/workflows/publish-to-pypi.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Publish Python distributions to PyPI and TestPyPI
+
+on:
+  push:
+    branches:
+    - 10-add-bumpversion
+    - main
```
[the remark this answers is not in the document]
Comment (cosunae) [reply .github/workflows/publish-to-pypi.yml:7]: I see, 
I think it is more straightforward to just trigger the action on tags: 
```
on:
  push:
    # Pattern matched against refs/tags
    tags:
```

[the remark this answers is not in the document]
Comment (victoria-cherkas) [reply .github/workflows/publish-to-pypi.yml]: changed to: 
on:
  push:
    tags:
    - *

**Comment by cosunae** [comment]
launch jenkins

**Pull request merged**